config/crd/bases/osp-director.openstack.org_openstackconfiggenerators.yaml

```diff
@@ -89,6 +89,11 @@ spec:
                 description: Required. the name of the config map containing Heat
                   env file customizations
                 type: string
+              heatEnvs:
+                description: List of heat environment files to include from tripleo-heat-templates/environments
```

Comment:
should we be here really specific that it is just the file name without any path?

Reply:
can include a path, but it's relative to t-h-t/environments

Reply:
e.g ssl/enable-internal-tls.j2.yaml

Reply:
oh right, sure. yes we need subnet references.